internal/connector/connector.go

```diff
@@ -57,6 +73,25 @@ type ServerOptions struct {
 	TrustedCertificate types.StringOrFile
 }
 
+func (o Options) APIClient() *api.Client {
+	url := o.Server.URL
+	if !strings.HasPrefix(url, "http") {
```

Comment:
Couldn't this result in
`https://example.com` -> `https://https://example.com`

Reply:
I don't think so, `https://...` also has a prefix of `http`, so that will skip this branch

Reply:
I've fixed this in the latest commit. I changed `options.Server.URL` to `cmd/types.URL` so that it's parsed earlier, and that lets us set the schema to `https` here without having to parse it.